src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js

```diff
@@ -15,11 +18,13 @@ type Props = {|
 
 const TaskCompletionRuleIcon = ({ completionRule }: Props): React.Node =>
     completionRule === TASK_COMPLETION_RULE_ANY && (
-        <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
-            <span>
-                <IconAnyTask height={11} width={11} className="bcs-TaskCompletionRuleIcon" />
-            </span>
-        </Tooltip>
+        <span className="bcs-TaskCompletionRuleIcon">
+            <LabelPill.Pill>
+                <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
+                    <LabelPill.Icon Component={IconAnyTask} />
```

Comment:
Using LabelPill.Icon to wrap IconAnyTask. Designs seem to have different height/width than what LabelPill.Icon provides. Not sure if this would need to be changed @booc0mtaco 
https://app.zeplin.io/project/5d437ee76e60a5450b0308b0/screen/5e44a99608ae50b49929001e